napari/layers/labels/labels.py

```diff
@@ -422,13 +422,13 @@ def rendering(self):
         """Return current rendering mode.
 
         Selects a preset rendering mode in vispy that determines how
-        lablels are displayed.  Options include:
+        labels are displayed.  Options include:
 
         * ``translucent``: voxel colors are blended along the view ray until
           the result is opaque.
         * ``iso_categorical``: isosurface for categorical data.
           Cast a ray until a non-background value is encountered. At that
-          location, lighning calculations are performed to give the visual
+          location, lightning calculations are performed to give the visual
```

Comment:
```suggestion
          location, lighting calculations are performed to give the visual
```
Typo but wrong fix

Reply:
what should it be? (sorry I am slow)

Reply:
I made the correction in the suggestion: lighting (like lights) and not lightning (like before thunder)